src/olympia/addons/management/commands/process_addons.py

```diff
@@ -29,7 +30,19 @@
     'add_firefox57_tag_to_webextensions': {
         'method': add_firefox57_tag,
         'qs': [Q(status=amo.STATUS_PUBLIC,
-                 _current_version__files__is_webextension=True)]}
+                 _current_version__files__is_webextension=True)]},
+    'bump_appver_for_legacy_addons': {
+        'method': bump_appver_for_legacy_addons,
+        'qs': [
+            Q(
+                type__in=(amo.ADDON_EXTENSION, amo.ADDON_THEME),
+                _current_version__files__is_webextension=False,
+                _current_version__apps__max__version_int__lt=56990000200100,
```

Comment:
This number so needs either a constant or a comment what it represents exactly. It's something <57, is that 56.* or so?